scripts/send_log_config.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Usage:
+    send_log_config.py new_log_cfg.conf 8888
+"""
+import json
+import socket
+import struct
+import sys
+
+cfg_file = sys.argv[1]
```

Comment:
For better maintainability and accessibility, I think no matter how insignificant an entry point is, it should have a good help text and an `argparse` interface.

Reply:
I've done some upgrades.